README.md

```diff
@@ -1,68 +1,102 @@
-# HTTP Movies
+# HTTP / AJAX II React Module Project: Movie CRUD
+
+This module explored HTTP methods, REST interfaces, CRUD apps and using put and delete to allow editing and deleting functionality. We also dug into how to pass props to Route connected components and using URL params to get values from an api. In this project, you will practice each of these skills by implement various pieces of functionality in a movie database CRUD app.
+
+## Objectives
+- Understand how to use the post, put and delete HTTP methods to interact with server data.
+- Understand how to sync server data with an applications internal state.
+- Understand how to pass values into a Route component to allow for the updating of state.
+
+## Introduction
+CRUD applications are the foundation of most web applications. Being able to manage creating, edit and deleting data from an external source is as key a skill as it gets. In this project, you will complete the code nessisary to allow all of these fundmental actions.
+
+![Movie DB Example](project-goals.gif)
+
+***Make sure to complete your tasks one at a time and complete test each task before proceeding forward.***
 
 ## Instructions
+### Task 1: Project Set Up
+* [ ] Create a forked copy of this project.
+* [ ] Clone your OWN version of the repository in your terminal
+* [ ] cd into the project base directory `cd web-module-project-reducer-pattern`
+* [ ] Download project dependencies by running `npm install`
+* [ ] Start up the app using `npm start`
+- [ ] With each saved change in your editor, the test runner will re-run all the tests
+- [ ] **IMPORTANT** If a test fails, use the test runner's error messages to find out why it is failing
 
-### Task 1: Project Setup 
+### Task 2: Project Requirements
+#### Editing a Movie
+> *Let's start by walking through the process of adding the routing, component and service calls need for resource updating*
 
-- **Fork** this repository, then clone your fork.
-- Run `npm install` to download dependencies.
-- Run the server using `npm start`.
-- In a separate terminal cd into the `client` folder and run `npm install` to download dependencies.
-- Still inside the `client` folder run `npm start` to run the client application.
+* [ ] First, we need to be able to navigate to the edit movie component. Add in the premade component used to edit a movie.
```

Comment:
I think we need to mention here something about routes. eg "Add the EditMovieForm component to the edit route in App.js"